server/command-line/storage.ts

```diff
@@ -65,4 +78,46 @@ function isUserLogEnabled(manager: ClientManager, user: string): boolean {
 	return conf.log;
 }
 
+async function runCleaning(user: string) {
+	const manager = new ClientManager();
+	const users = manager.getUsers();
+
+	if (user) {
+		if (!users.includes(user)) {
+			throw new Error(`invalid user ${user}`);
+		}
+
+		return cleanUser(manager, user);
+	}
+
+	for (const name of users) {
```

Comment:
I'm confused by this logic. We return above if the user is defined, and if no user is passed we clean all users?

Reply:
yes. The command defaults to all users, you can however run a specific one if you need to for whatever reason.

Usually the admin wants to migrate all users at the same time... so defaulting to that makes sense no?

Reply:
See suggestion above